eternalcombat-api/src/main/java/com/eternalcode/combat/notification/NotificationAnnouncer.java

```diff
@@ -69,6 +74,26 @@ public void broadcast(String text) {
         audience.sendMessage(message);
     }
 
+    public void sendMessage(Player player, String text, Duration timeLimit) {
```

Comment:
You can just copypaste from EternalCore  Delay class end hold this in listener or specific manager for that